src/current/cockroachcloud/cloud-api.md

```diff
@@ -669,16 +799,17 @@ The service account associated with the secret key must have the Cluster Adminis
 {% include_cached copy-clipboard.html %}
 ~~~ shell
 curl --request DELETE \
-  --url 'https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/sql-users/{sql_username}'' \
+  --url 'https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/sql-users/{sql_username}' \
   --header 'Authorization: Bearer {secret_key}'
 ~~~
 
 Where:
 - `{cluster_id}` is the unique ID of this cluster.
 {{site.data.alerts.callout_info}}
-The cluster ID used in the Cloud API is different from the routing ID used when [connecting to clusters]({% link cockroachcloud/connect-to-a-basic-cluster.md %}).
+The cluster ID used in the Cloud API is different from the routing ID used when [connecting to clusters]({% link cockroachcloud/connect-to-your-cluster.md %}).
 {{site.data.alerts.end}}
 - `{sql_username}` is the username of the SQL user you want to delete.
+- `{secret_key}` is the secret key for the service account.
 
 If the request was successful, the client will receive a response with the name of the deleted user.
 
```

Comment:
I don't think we need to show the output.

Reply:
Refined like the others